### PR TITLE
Support in uint64 added

### PR DESCRIPTION
**Pull request by Bezbran**
Fix of the following issue:
#71

### CODE REVIEW

**Comment by Parkayun** [comment]
Awesome job!
Could you add some tests?

**Comment by Bezbran** [comment]
Done.
@Parkayun

File: bson/tests/test_int.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+from unittest import TestCase
+
+from bson import dumps, loads
+
+
+class TestInt(TestCase):
+    def setUp(self):
+        self.goodRquestDict = {
```
Comment (Parkayun) [inline bson/tests/test_int.py]: Could you change to use underscore `good_request_dict`?
I will merge after this! :)

**Comment by Bezbran** [comment]
You are right. My fault.
Done.

File: bson/tests/test_int.py (hunk 1 above)
Comment (Bezbran) [reply bson/tests/test_int.py]: Done.
@Parkayun

**Comment by Parkayun** [comment]
Thanks!

**Pull request merged**